arch/arm/Kconfig

```diff
@@ -1289,6 +1289,7 @@ config UNWINDER_STACK_POINTER
 
 config UNWINDER_ARM
 	bool "ARM EHABI stack unwinder"
+	select FRAME_POINTER
```

Comment:
I'm only not sure of select vs depend here

Reply:
I think that select fits better to avoid unset dependenyes